app/models/task_management.rb

```diff
@@ -24,7 +24,7 @@ def self.notifications_count(user_type, id)
 
   def self.all_notifications_for(user_type, id)
     if user_type == "taskee"
-      where(taskee_id: id, status: "inactive").
+      where(taskee_id: id, status: "inactive", paid: true).
```

Comment:
This is because we will not want a taskee to see assigned tasks that have not yet been paid for.